pyuvdata/parameter.py

```diff
@@ -846,6 +846,10 @@ class LocationParameter(UVParameter):
         Description of the data or metadata in the object.
     frame : str, optional
         Coordinate frame. Valid options are "itrs" (default) or "mcmf".
+    lunar_ellipsoid : str, optional
```

Comment:
Is it worth generalizing this to just `ellipsoid`? ITRF of course has different realizations, and  it may be helpful for this keyword to remain flexible with other frames as well (e.g., when we eventually support interferometers on Pluto in pyuvdata v9.0). Supporting this for ITRF frames may require some retooling of defaults (since "SPHERE" obviously is not the desired ellipsoid for Earth), but I think treating this as a programmable parameter for all but ITRF would help provide some future-proofing.

Reply:
Sure, I wondered about that actually. What's the right default for ITRS?

Reply:
Maybe `None` is the correct thing for ITRS?

Reply:
So I think either `None` or `"WGS84"` (the astropy default) might be sensible -- there are some subtleties here that I have yet to dig into, but I think either would be a reasonable choice while we investigate further.